django/urls/resolvers.py

```diff
@@ -317,6 +318,9 @@ def __init__(self, pattern, callback, default_args=None, name=None):
         self.default_args = default_args or {}
         self.name = name
 
+    # def __str__(self):
```

Comment:
A test is missing to show why this change belongs in the PR.

Reply:
Good point, it belonged in #9323 but is not needed anymore: I removed it.